src/server/routing/__tests__/event.router.spec.ts

```
@@ -11,9 +11,10 @@ import { MessageProvider } from '../../infrastructure';
 
 import { RpcContext } from '../../../context';
 import { StreamKind } from '../../../interfaces';
-import type { DeadLetterConfig, EventProcessingConfig } from '../../../interfaces';
+import type { DeadLetterConfig, EventProcessingConfig, JetstreamModuleOptions } from '../../../interfaces';
```

ESLint

[error] Replace `·DeadLetterConfig,·EventProcessingConfig,·JetstreamModuleOptions·` with `⏎··DeadLetterConfig,⏎··EventProcessingConfig,⏎··JetstreamModuleOptions,⏎`

(prettier/prettier)